### DISCUSSION TITLE
Feedback on the debugger

### DISCUSSION THREAD

**Discussion by tommasongr** [nova-ruby/debug#1] [Q&A]
## 👋 Ciao!

This is the place to collect all the feedback, experiences, ideas, etc on the debugger.

In particular I would love to have some feedback on:

- remote debugger
- remote debugger configuration with projects other than Rails
- setup and documentation
- limitations and other issues

Thanks to all in advance!

**Comment by svoop** [comment]
Great news, thanks for your work on this!

A quick try of the remote debugger worked just fine. I ran into a problem when the debugger crashed, but this could be an issue of my code – I'll have to try and reproduce. Will have more time this weekend.
